highlevel/robot/router.py

```diff
@@ -43,6 +48,11 @@ async def dispatch_message(self, bus_message: BusMessage,
             LOGGER.get().info("low_level_log",
                               content=bus_message.debugLog.content,
                               source=source)
+        elif type_msg in [
```

Comment:
at least add a comment on why they are ignored

also you can extract that list and convert it to a set for O(1) access